assets/airports/kmsy.json

```diff
@@ -0,0 +1,2833 @@
+        "IRUBE": ["N31d00m15.9500", "W88d56m18.6200"],
+        "PAYTN": ["N31d28m04.3500", "W87d53m07.9100"],
+        "MUURY": ["N30d45m55.2000", "W87d44m31.4600"],
+        "INBRD": ["N30d48m29.9600", "W87d00m27.1100"],
+        "CEW": [30.8261739, -86.6791433],
+        "DEFUN": ["N30d48m51.2700", "W86d07m53.0900"],
+        "MCGEE": ["N31d43m19.2300", "W90d15m42.7000"],
+        "SQS": [33.4638494, -90.2773506],
+        "ACMES": ["N30d55m27.1300", "W88d22m10.8200"],
+        "BUNNZ": ["N28d00m34.9100", "W86d45m45.5200"],
+        "BACCA": ["N27d35m30.5300", "W85d20m39.6000"],
+        "VUZ"  : [33.6701322, -86.8998372],
+        "MEM"  : [35.0151058, -89.9832017],
+        "TWOUP": ["N33d53m45.390", "W83d49m08.390"],
+        "SPA"  : [35.0336258, -81.9270100]
+    },
+    "restricted": [],
+    "runways": [
+        {
+            "name": ["2", "20"],
+            "end": [				
+                ["N29d59.070092", "W90d15.084900", "1.8ft"],
+                ["N30d00.183207", "W90d14.730605", "-2.4ft"]
+            ],
+            "ils": [true, false]
+        },	
+        {
+            "name": ["11", "29"],
+            "end": [				
+                ["N29d59.797593", "W90d16.903735", "3.7ft"],
+                ["N29d59.352757", "W90d15.058157", "1.3ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "airways": {
+        "A770": ["LEV", "DOLPH", "ALGAE"],
+        "J2"  : ["LCH", "LSU", "SJI", "MUURY", "INBRD", "CEW", "DEFUN"],
+        "J31" : ["LEV", "HRV", "MEI"],
+        "J58" : ["HRV", "PEBBY", "AEX"],
+        "L214": ["LEV", "PLNDR", "DAGGR"],
+        "L333": ["HRV", "HOOCK", "TRESR"],
+        "M345": ["TBD", "WAHOO", "KENGS"],
+        "Q22" : ["OYSTY", "ACMES", "CATLN", "TWOUP", "SPA"],
+        "Q24" : ["LSU", "IRUBE", "PAYTN"],
+        "Q56" : ["PEKON", "HRV", "SJI", "CATLN"],
+        "Q105": ["HRV", "FATSO", "REDFN"],
+        "V9"  : ["LEV", "PEPSI", "GOING", "SAFES", "RAYOP", "WAVEZ", "OYSTY", "RYTHM", "FOILS", "ZYDCO", "MCB"],
+        "Y280": ["REMIS", "ROZZI", "NAITE", "REDFN", "LEV"],
+        "Y290": ["LEV", "BLVNS", "BUNNZ", "BACCA"]
+    },
+    "sids": {},
+    "stars": {
+       "AWDAD1": {
+            "icao": "AWDAD1",
+            "name": "Awdad One",
+            "entryPoints": {
+                "LCH": ["LCH", "LFT"],
+                "LFT": ["LFT"],
+                "LSU": ["LSU"]
+            },
+            "body": ["AWDAD", "VOODO"],
+            "rwy": {
+                "KMSY2": [],
+                "KMSY11": [],
+                "KMSY20": [],
+                "KMSY29": []
+            },
+            "draw": [
+                ["LSU", "AWDAD", "LFT", "LCH"],
+                ["AWDAD", "VOODO"]
+            ]
+        },
+        "OLEDD4": {
+            "icao": "OLEDD4",
+            "name": "Oledd Four",
+            "entryPoints": {
+                "LLA": [],
+                "TBD": []
+            },
+            "body": ["LLA", "ORICH", "LUFFS", "TBD", "OLEDD"],
```

Comment:
This won't work since TBD is a possible entryPoint

Reply:
fixed